src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -9700,29 +9740,50 @@ let rec EvalAttribArgExpr g x =
     // Detect bitwise or of attribute flags
     | AttribBitwiseOrExpr g (arg1, arg2) -> 
```

Comment:
Maybe this can be SpecificBinopExpr too?

Reply:
Don't think so, there's some extra logic there
![image](https://user-images.githubusercontent.com/5063478/203328620-310134f6-86eb-424f-92a5-f1c53c5fbcc1.png)